test/test_external.py

```diff
@@ -1,18 +1,16 @@
 import os
 from tempfile import gettempdir
 
-from pyabc.external import R
 import pyabc
-
-
-r_file = "doc/examples/myRModel.R"
+import pyabc.external
 
 
 def test_r():
     """
     This is basically just the using_R notebook.
```

Comment:
I would rather say "this is reproducing the using_R notebook" (?) because is not exactly the same from a coding point of view.